### DISCUSSION TITLE
How to use [CssClass("in here my class")] in my forms???

### DISCUSSION THREAD

**Discussion by ra-bedo11** [serenity-is/Serenity#6358] [Help]
public class TblCustomerForm
    {
        [Category("Basic")]
        [CssClass("in here my class")]
        public String Name { get; set; }

    }
}
this not work why ????

**Comment by dfaruque** [comment]
this is working. check with browser inspector. the class is applied to the `<div class="field Name">` element

**Reply by ra-bedo11** [reply]
I Put this code in site.rtl.css

.xxx{
    max-width:30%;
}
and in forms put
    [CssClass("xxx")]
    public String Name { get; set; }

this not work ? when choose Page and Language  ( RTL - Arabic )

**Comment by ra-bedo11** [comment]
?????
